.pre-commit-config.yaml

```diff
@@ -3,19 +3,19 @@ repos:
   rev: v2.31.0
   hooks:
   - id: pyupgrade
-    args: [--py36-plus]
+    args: [--py37-plus]
 - repo: https://github.com/python/black
-  rev: 22.1.0
+  rev: 22.3.0
   hooks:
   - id: black
     language_version: python3
 - repo: https://gitlab.com/pycqa/flake8
   rev: 3.9.2
   hooks:
   - id: flake8
-    additional_dependencies: [flake8-bugbear==22.1.11]
+    additional_dependencies: [flake8-bugbear==22.1.11, importlib-metadata==4.12.0]
```

Comment:
Because flake8 needed an older version. I didn’t want to update flake8 to keep the PR scoped.
Thanks for maintaining the package!
I see you have updated the flake8 version to resolve this